modules/images-other-jenkins-monitored.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins.adoc
+
+[id="images-other-jenkins-monitoring_{context}"]
+= Jenkins Monitoring
+
+When deployed by the provided Jenkins Monitored templates, 
+Jenkins is configured to expose its metrics to Prometheus.
+
+Note: This template requires user to have permission to create `servicemonitor` 
```

Comment:
Since this is a procedure, add: 

(if there are prerequisites)
.Prerequisites

* [List any prereqs]

.Procedure

. This template requires the user to have permission to create `servicemonitor` objects in the namespace where Jenkins needs to be deployed. You can create a specific `clusterrole` and bind it to the user: 
+
---- (four hyphens, the code block is going to render wrong on Github)
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: servicemonitor-edit
  labels:
    rbac.authorization.k8s.io/aggregate-to-admin: "true"
    rbac.authorization.k8s.io/aggregate-to-edit: "true"
rules:
- apiGroups: ["monitoring.coreos.com"]
  resources: ["servicemonitors"]
  verbs: ["create", "update", "patch"]
---- (four hyphens) 
+
. Add this role to user in the targeted namespace:
+
---- (four hyphens)
$ oc adm policy add-role-to-user servicemonitor-edit user -n namespace
---- (four hyphens)